File: Array/Contains Duplicate/Contains_Duplicate.cpp
class Solution
{
public:
    bool containsDuplicate(vector<int> &nums)
    {
        unordered_map<int, bool> um;
        for (int i = 0; i < nums.size(); i++)
        {
            if (um[nums[i]] == true)
                return true;
            else
                um[nums[i]] = true;
        }
        return false;
    }
};

File: Array/Contains Duplicate/Contains_Duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        c=collections.Counter()
        c.update(nums)
        for i in c:
            if(c[i]>1): return True
        return False